server/routes/api/repos.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const { updateRecord } = require('./../../gitUtility/initialSeed');
+const { Repository } = require('./../../db/repositories');
+
+const router = express.Router();
+
+// Middleware used to check if authenticated.
+const isAuthenticated = (req, res, next) => {
+    if (req.isAuthenticated()) {
+        return next();
+    }
+    console.log('isAuthenticated: No');
+    res.redirect('/user/login');
+    return 0;
+};
+
+router.get('/v1/search', (req, res) => {
+    const query = {};
+    if (req.query.stargazer) {
+        query.stargazersLogin = { $all: [req.query.stargazer] };
+    }
+    // Filter language depending on what selected.
+    if (req.query.language && req.query.language !== 'Any') {
```

Comment:
`Any` && `NoLanguage` can live in their own file, something like `constants/index.js`